src/DDoSX/DomainPropertiesClient.php

```diff
@@ -0,0 +1,45 @@
+<?php
+namespace UKFast\SDK\DDoSX;
+
+use UKFast\SDK\Client as BaseClient;
+use UKFast\SDK\DDoSX\Entities\DomainProperty;
+use UKFast\SDK\SelfResponse;
+
+class DomainPropertiesClient extends BaseClient
+{
+    protected $basePath = 'ddosx/';
+
+    const DOMAIN_PROPERTIES_MAP = [];
+
+    /**
+     * Update a Domain's property
+     * @param $domainId
+     * @param $propertyId
+     * @param $data
+     * @return SelfResponse
+     * @throws \GuzzleHttp\Exception\GuzzleException
+     */
+    public function update($domainId, $propertyId, $data)
```

Comment:
in other places, we pass it the object being updated (sometimes with, sometimes without the `id` separately) - so possibly use `update($domainName, DomainProperty $property)` or just rename the $data param?